website/static/js/share/index.js

```diff
@@ -36,9 +38,16 @@ ShareApp.view = function(ctrl) {
             SearchBar.view(ctrl.searchBarController),
             Stats.view(ctrl.statsController),
             m('br'),
-            Results.view(ctrl.resultsController),
+            m('.row', [
+               m('.col-md-2', [
+                    SideBar.view(ctrl.sideBarController)
+                ]),
+                m('.col-md-10', [
+                    Results.view(ctrl.resultsController)
+                ])
+            ]),
             m('.row', m('.col-md-12', {style: 'padding-top: 30px;'}, m('span', m.trust(MESSAGES.ABOUTSHARE))))
-        ])
+        ]),
```

Comment:
Trailing commas not not recommended in javascript, its confusing if anything is supposed to be following or not when in deeply nested structures.